addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22485,3 +22485,24 @@ msgstr ""
 msgctxt "#39153"
 msgid "Windowing system:"
 msgstr ""
+
+#: system/settings/settings.xml
```

Comment:
Please add a verbal description (#. text) to help the translators, here and for the other new UI strings.